throttle/contract_tx_limiter_middleware.go

```diff
@@ -144,6 +198,9 @@ func NewContractTxLimiterMiddleware(cfg *ContractTxLimiterConfig,
 			}
 			txl.contractToTierMap = contractToTierMap
 			txl.contractDataLastUpdated = time.Now().Unix()
+			contractTierMapLoadDuration := time.Since(start)
+			lvs := []string{"method", "contractTierMap Load", "error", fmt.Sprint(err != nil)}
+			contractTierMapLoadLatency.With(lvs...).Observe(contractTierMapLoadDuration.Seconds())
```

Comment:
Move the code in this if block into a separate function and instrument that function, otherwise the error won't be captured.

Reply:
Done.